pos_self_service_weighing_tare/static/src/xml/Screens/SelfServiceWeighingTareScreen/SelfServiceWeighingTareScreen.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+SPDX-FileCopyrightText: 2024 Coop IT Easy SC
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<templates id="template" xml:space="preserve">
+
+    <t t-name="pos_self_service_weighing_tare.SelfServiceWeighingTareScreen" owl="1">
+        <div class="screen">
+            <div class="screen-full-width">
+                <div class="leftpane pane-border">
+                    <WeightWidget />
+                    <div class="self-service-control-buttons">
+                        <SelfServiceWeighingHomeButton />
+                    </div>
+                </div>
+                <div class="rightpane">
+                    <div class="self-service-action-buttons">
+                        <SelfServiceWeighingPrintTareButton />
+                    </div>
+                </div>
+            </div>
+        </div>
+    </t>
```

Comment:
This completely re-implements `pos_self_service_weighing_base.AbstractSelfServiceWeighingScreen`, which I'm not sure is intentional. Can you inherit the latter template without changing it for other inheritors?

Reply:
Talked with @robinkeunen about this. We can do primary inheritance here as described in my previous comment, _but_ there's a chance that the primary inheritance would be more lines of code in a much less transparent way. e.g. insert a button here, insert a button there, etc.

For developer ease, it may be nice to just see the simple hierarchy like in the quoted code. The hierarchy is sufficiently small that the repetition is barely a problem. And there would be repetition, because there will be other screens that re-implement `pos_self_service_weighing_base.AbstractSelfServiceWeighingScreen`. Furthermore, although we _currently_ think that other implementations of `AbstractSelfServiceWeighingScreen` will look incredibly similar, this may not always be the case.

But… For the time being, it's still repetition. And because this hierarchy kind of depends on `pos.css` from the main module to display well, it's probably better to do primary inheritance?

Your thoughts @huguesdk ?

Reply:
@huguesdk we discussed this with Carmen. It is now obvious to us wether primary inheritance is better (less code - maybe) or worse (lacks clarity) then redefining a template. What is your point of view ?